src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java

```diff
@@ -54,6 +55,30 @@ public static void main(String[] args) {
 		SoundEffect.BGM.play();
 	}
 
+	// Set up buttons
+	public void buttonOperation(JButton b, Color c){
+		b.setOpaque(false);
+		b.setContentAreaFilled(false);
+		b.setBorder(BorderFactory.createLineBorder(c));
+		b.setForeground(Color.white);
+		b.setFont(new Font("Century Gothic", Font.PLAIN, 18));
+	}
+
+	// Mouse listener for the hover effect
+	public void buttonAddListener(JButton b, Color c1, Color c2, Color c3, Color c4){
```

Comment:
Should be private.

Reply:
I'm not convinced we need four colors here.  Maybe only three (or two if we can assume the default foreground color is always white)?